plugins/woocommerce/src/Blocks/BlockTypes/AccordionGroup.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+/**
+ * AccordionGroup class.
+ */
+class AccordionGroup extends AbstractBlock {
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'accordion-group';
+
+	/**
+	 * Include and render the block.
+	 *
+	 * @param array    $attributes Block attributes. Default empty array.
+	 * @param string   $content    Block content. Default empty string.
+	 * @param WP_Block $block      Block instance.
+	 * @return string Rendered block type output.
+	 */
+	protected function render( $attributes, $content, $block ) {
+		if ( ! $content ) {
+			return $content;
+		}
+	
+		$suffix = wp_scripts_get_suffix();
+		if ( defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN ) {
+			$module_url = gutenberg_url( '/build-module/block-library/accordion-group/view.min.js' );
+		}
```

Comment:
Is this necessary?

Reply:
No, I can remove it!
Although if the accordion does end up being merged to GB, we did have to implement something similar I think.